drivers/input/i8042prt/keyboard.c

```diff
@@ -344,8 +344,10 @@ i8042KbdDpcRoutine(
 	}
 
 	i8042PacketDpc(PortDeviceExtension);
+
```

Comment:
The newlines weren't broken, they don't need fixing.

Reply:
Done.

Reply:
Thanks